themes/10up-theme/assets/css/frontend/style.css

```diff
@@ -4,13 +4,40 @@
  *
```

Comment:
In general, is this the specificity order of "layers" in Tailwind? I'm fairly unversed in TW and layers are new for myself. I imagine other Engineers might have the same inexperience? I wonder if we can expand on this methodology a bit to help other engineers?

`@tailwind base;
@tailwind components;
@tailwind utilities;`

Reply:
In other words, in terms of layers, utilities would take priority here, correct?

Reply:
@devinle Correct, because it is last in the layer order (I believe tailwind is using `@layer` under the hood when it calls `@tailwind`). [See their docs on @layer](https://tailwindcss.com/docs/adding-custom-styles#using-css-and-layer) and this quote from that section (emphasis mine):

> For more power, you can also use the `@layer` directive to **add styles to** Tailwind’s base, components, and utilities layers...

Use of `@layer` is still pretty new for everyone, so I'm down having more documentation here.

I get the feeling we'd mix this with `@import` like so:

In your index:
```css
@import url('foo.css');
```

In the file:
```css
@layer components {
  .foo {
    /* some styles... */
  }
}
```

Reply:
@devinle @Firestorm980 on my todo list to add more documentation here with some examples! Great feedback!

Reply:
@Nikki-Jones - that would be great, I think this is a good point to bring up!

Reply:
@Firestorm980 @devinle added some doc updates here for you: https://github.com/10up/wp-scaffold/pull/162